### PR TITLE
Fix: [KRAANG-7741] PlanGrid: when viewing photos from documents and using arrows to go to next photo the image doesn't change 

### PR DESCRIPTION
**Pull request by akinnee**
The photo viewer component never handled the prop changing. It appears this has never worked.

I've also upgraded a couple dependencies just to make this repo install & run on modern developer hardware & allow using hooks in the demo component.

 JIRA Tickets | 
 --------------| 
 [KRAANG-7741](https://jira.autodesk.com/browse/KRAANG-7741)|

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -20,11 +20,50 @@ import mp3 from '../example_files/sample.mp3'
...
 
+const demoImages = [solarImage, photo360];
+
+const getNextDemoImage = (currentDemoImage) => {
+  const currentIndex = demoImages.indexOf(currentDemoImage);
+  const nextIndex = currentIndex + 1 >= demoImages.length ? 0 : currentIndex + 1;
+  return demoImages[nextIndex];
+};
+
+const App = () => {
```
Comment (akinnee) [inline src/app.js:31]: This is just the demo that runs when we run `npm run start`.

File: src/components/drivers/photo-viewer-wrapper.jsx [hunk 2]
```diff
@@ -26,15 +26,31 @@ export default class PhotoViewerWrapper extends Component {
   }
 
   componentDidMount() {
-    // spike on using promises and a different loader or adding three js loading manager
+    this.loadPhoto();
```
Comment (akinnee) [inline src/components/drivers/photo-viewer-wrapper.jsx:29]: All the photo loading code only ran here, on mount. I moved it to loadPhoto, which also runs if the filePath prop changes now.

File: webpack.config.dev.js [hunk 3]
```diff
@@ -63,6 +63,9 @@ module.exports = {
           },
           {
             loader: 'sass-loader',
+            options: {
+              implementation: require('sass'),
```
Comment (akinnee) [inline webpack.config.dev.js:67]: node-sass is deprecated, and the old version that was installed here will not compile on arm64. node-sass recommends using this sass package instead.

**Review by drewbrend — COMMENTED** [review]
Is the massive pacakge-lock.json diff expected?

**Comment by akinnee** [comment]
> Is the massive pacakge-lock.json diff expected?

Yes, it's because we've moved from jfrog to npm.autodesk.com.

**Comment by akinnee** [comment]
> > Is the massive pacakge-lock.json diff expected?
> 
> Yes, it's because we've moved from jfrog to npm.autodesk.com.

Oh, and we've also upgraded from lockfile v1 to v2. That is necessary to use a reasonably recent version of npm at this point.

File: README.md [hunk 4]
```diff
@@ -72,7 +72,7 @@ purposes. It is by default served via webpack-dev-server.
 
 ### To start demo app
 
-`make start` will start the demo app served by webpack-dev-server
```
Comment (drewbrend) [inline README.md:75]: Can the makefile be removed now or is it still required?

Comment (akinnee) [reply README.md:75]: I was confused why there's a makefile at all. Removing it is out of scope for this PR though.

**Review by drewbrend — APPROVED** [review]

**Comment by akinnee** [comment]
Thanks for the review @drewbrend.

**Pull request merged**